src/server/cli/test.js

```diff
@@ -0,0 +1,159 @@
+import { ConfigLoaderError, readConfig } from '@web/config-loader';
+import commandLineArgs from 'command-line-args';
+import commandLineUsage from 'command-line-usage';
+import process from 'node:process';
+import { startTestRunner } from '@web/test-runner';
+import { WTRConfig } from '../wtr-config.js';
+
+async function getTestRunnerOptions(argv = []) {
+
+	const DISALLOWED_OPTIONS = ['--browsers', '--playwright', '--puppeteer', '--groups'];
+
+	const optionDefinitions = [
+		// @web/test-runner options
+		{
+			name: 'files',
+			type: String,
+			multiple: true,
+			description: 'Test files to run. Path or glob.\n[Default: ./test/**/*.<group>.js]',
+			order: 8
+		},
+		{
+			name: 'group',
+			type: String,
+			required: true,
+			defaultOption: true,
+			description: 'Name of the group to run tests for\n[Default: test]',
+			order: 1
+		},
+		{
+			name: 'manual',
```

Comment:
Not sure if I raised this here or the other PR, this will still be removed in a future PR?